app/views/measures/grouped/_xi.html.erb

```diff
@@ -4,11 +4,18 @@
 <!-- CUSTOMS DUTIES -->
 <% if xi_import_measures.customs_duties.present? %>
   <%= render 'measures/meursing_form' if xi_declarable.meursing_code? %>
-  <%= render partial: 'measures/grouped/table', locals: {
-    caption: 'Customs duties',
-    collection: xi_import_measures.customs_duties.sort_by(&:key),
-    css_id: 'customs_duties' }
-  %>
+
+  <h3 class="govuk-heading-m govuk-!-margin-top-7 govuk-!-margin-bottom-3", id='customs_duties'>
+    Customs duties
+  </h3>
+
+  <div class="govuk-inset-text govuk-inset-text--s nhs-grey-5 explainer">
```

Comment:
We should put this into a partial, really. Lots of duplication here

Reply:
👍  refactored!